pkg/kubelet/kubelet.go

```diff
@@ -793,7 +794,11 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 
 	klet.appArmorValidator = apparmor.NewValidator(kubeCfg.ContainerRuntime)
 	klet.softAdmitHandlers.AddPodAdmitHandler(lifecycle.NewAppArmorAdmitHandler(klet.appArmorValidator))
-
+	if utilfeature.DefaultFeatureGate.Enabled(features.Accelerators) {
+		klet.gpuManager = nvidia.NewNvidiaGPUManager(klet, klet.dockerClient)
```

Comment:
What happens if the runtime is not docker? Should probably check if `--container-runtime`  is docker and error out if it's not.